### PR TITLE
Feedback testing on OpenStack Ubuntu deployment

### PR DESCRIPTION
**Pull request by zonca**
Still work in progress, just wanted to open this in case I don't come back to this for a while...

I'm testing the ansible deployment on our OpenStack at SDSC on Ubuntu 16.04, sending feedback as I go.

### CODE REVIEW

**Comment by willingc** [comment]
@zonca Thanks! :cookie:

**Comment by minrk** [comment]
Great, thanks!​

**Comment by zonca** [comment]
@willingc @minrk 

completed my testing, the biggest issue I found is related to the conda path, opened an issue about that https://github.com/jupyterhub/jupyterhub-deploy-teaching/issues/20, I think it would be interesting to investigate that more.

the other change is that I set ansible up to use the `ubuntu` user instead of `root`, I believe this is the default setup for new ubuntu installations, it should make it easier for users to deploy this.

Then I have other minor changes,
please let me know what you are interested in merging and what not so I can improve the pull request.

**Comment by zonca** [comment]
fixed #20 with https://github.com/jupyterhub/jupyterhub-deploy-teaching/pull/19/commits/a058eadda3ce8217c60a78211359a92cefee698a

Also automatically installing python 2.7 on the remote machine now.

File: docs/source/installation.rst [hunk 1]
```diff
@@ -6,19 +6,23 @@ Prerequisites
...
 - Verify a valid DNS entry for the server.
 
 - Choose an SSL certificate source. Use either of these options:
 
   * `Let's Encrypt <https://letsencrypt.org/>`_
   * obtain a trusted SSL certificate and key for the server at that FQDN.
 
+- Checkout the latest version of the repository including the ``ansible-conda`` submodule::
+
+    $ git clone https://github.com/jupyterhub/jupyterhub-deploy-teaching.git
```
Comment (minrk) [inline docs/source/installation.rst]: The easiest way to do this is `git clone --recursive`, rather than three commands.

**Comment by minrk** [comment]
Excellent, thanks @zonca!

**Comment by zonca** [comment]
thanks @minrk , do you want me to split this PR in smaller more focused PRs?

**Comment by minrk** [comment]
Nope, I think this one is AOK as-is. Is there anything else you want to cover here before merge (since it's still marked WIP)? If not, I'm happy to merge this now.

**Comment by willingc** [comment]
@zonca I agree with @minrk. We're happy to merge if this is no longer WIP. Thank you!

**Comment by zonca** [comment]
Great, yes, I'm done

**Comment by willingc** [comment]
Thanks @zonca :cake: :cookie:

**Pull request merged**